modules/virt-vm-creating-nic-web.adoc

```diff
@@ -21,4 +21,4 @@ You can configure a network interface for a virtual machine (VM) by using the {p
 . Click *Add network interface*.
 . Enter the interface name and select the network attachment definition from the *Network* list.
 . Click *Save*.
-. Restart the VM to apply the changes.
+. Restart or live-migrate the VM to apply the changes.
```

Comment:
We don't hyphenate "live migrate" (though I do see there is one instance in the docs, which we should fix later).